docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,71 @@
+# Release 403 (15 Nov 2022)
+
+## General
+
+* Include the amount of data read from external sources in the output of
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Improve performance of worker-to-worker data transfer encryption when
+  fault-tolerant execution is enabled. ({issue}`14941`)
+* Improve performance of aggregations when input data does not contain nulls. ({issue}`14567`)
+* Fix potential failure when clients do not support variable precision temporal
+  types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+* Fix failure for queries involving `BETWEEN` predicates over `varchar` columns
+  that contain temporal data. ({issue}`14954`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 authentication with
+  refresh tokens enabled. ({issue}`14949`)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a metadata cache. This can be
+  configured with the `bigquery.metadata.cache-ttl` catalog property, which is
+  disabled by default. ({issue}`14729`)
+* Fix failure when a [row access policy](https://cloud.google.com/bigquery/docs/row-level-security-intro)
+  returns an empty result. ({issue}`14760`)
+
+## ClickHouse connector
+
+* Add mapping for the ClickHouse `DateTime(timezone)` type to the Trino
+  `timestamp(0) with time zone` type for read-only operations. ({issue}`13541`)
+
+## Delta Lake connector
+
+* Fix statistics for `DATE` columns. ({issue}`15005`)
```

Comment:
We should remove this.  https://github.com/trinodb/trino/pull/15005 has no practical end-user implications that I am aware of, unless "bechmarks" is a "user".

cc @raunaqmorarka

Reply:
Well .. isn't this a performance improvement? If so .. there is user facing impact.

Reply:
Not in any practical situation, so no, it's not user visible.

Reply:
Okay .. then I am not understanding this. Whats the point of having these stats then.. or fixing them. If there is no impact.

Reply:
The change is about fixing stats for DATE in a particular range (1900-01-02 to 1901-12-31). This range occurs in TPCDS and not having accurate stats results in bad plans for certain TPCDS queries. 
I have no idea how to assess if this is possible in user data, however we know that ppl run TPC benchmarks and care about it running faster.

Reply:
in any case, "Fix statistics for `DATE` columns" is bad wording, since nothing was broken and nothing got fixed.

you can change this to 

"Add support for date column min/max statistics for values in range 1900-01-02 between 1901-12-31. Such dates occur in TPC-DS data sets."


This would be fathful, but also would tell the truth that real users don't care about this change, because no production workload will benefit. No-one runs TPC-DS as "production workloads".

Reply:
My suggested RN in the PR was `Improve statistics for DATE columns in delta lake`.
It's inaccurate to claim that no production workload can ever benefit from it. How do you know that there isn't some dimension table in "real" data which contains birth dates of ppl from that time ?

Reply:
> It's inaccurate to claim that no production workload can ever benefit from it. 

Lack of release notes entry doesn't claim that.

Existence of release notes entry claims that there is a benefit.

> How do you know that there isn't some dimension table in "real" data which contains birth dates of ppl from that time ?

How do you know there is?

Reply:
I agree with the rewording to make what was fixed more explicit. We should reserve judgement on whether such data appears in production workloads. That is very situational. There are many domains that deal with data in that range (and older) - historical weather record, etc.

Reply:
For applications dealing with real historical data the fix won't really help, as it moves the min supported min value by just a year.